pkg/ccl/jobsccl/jobs_info_storage_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package jobsccl
```

Comment:
we _are_ allowed to make bsl test -> ccl deps these days so I'm not sure this new package is strictly necessary

Reply:
old habits die hard, done!